src/sortStudents.ts

```diff
@@ -1,16 +1,62 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function getAvarageGrade(grades: number[]): number {
+  return grades.reduce((a, b) => a + b, 0) / grades.length;
```

Comment:
Better use meaningful variable names
 Rename this way according to [method](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)
```suggestion
  return grades.reduce((previous, current) => previous + current, 0) / grades.length;
```

Or better more specific for this task
```suggestion
  return grades.reduce((sum, grade) =>sum + grade, 0) / grades.length;
```